ucb-rit/coldfront
# fix #14

#14 

Changes:
- Added info messages when Managers try to demote themselves
- Added constraints matching the discussion post below

How to test:
- be a manager on a project, and try to deactivate notifications. the only way to do so is by demoting yourself
- be a pi and try to edit your own notification settings, the constraints should match

Discuss:
- a pi an deactivate notifications of other users... if their own were already disabled, this will leave no users with notifications on, other than manager... is this behavior okay?
